packages/paste-website/src/components/empty-state/NotBuilt.tsx

```diff
@@ -1,10 +1,12 @@
 import * as React from 'react';
 import {Anchor} from '@twilio-paste/anchor';
-import {Text} from '@twilio-paste/text';
 import {Box} from '@twilio-paste/box';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Heading} from '@twilio-paste/heading';
```

Comment:
Should we test `import {Heading, Paragraph} from '@twilio-paste/typography';` to verify its working as expected?

Reply:
smart